### PR TITLE
Admin interface for listing, adding and removing custom emojis

### PR DESCRIPTION
**Pull request by Gargron**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -108,6 +108,18 @@ en:
       unsubscribe: Unsubscribe
       username: Username
       web: Web
+    custom_emojis:
+      created_msg: Emoji successfully created!
+      delete: Delete
+      destroyed_msg: Emojo successfully destroyed!
```
Comment (lynlynlynx) [inline config/locales/en.yml]: Typo 😭

Comment (Gargron) [reply config/locales/en.yml]: I did it on purpose as a nod to a popular joke on Mastodon. It's a rare occuring string, should be fine

Comment (lynlynlynx) [reply config/locales/en.yml]: Ah, convinced 👍

File: app/controllers/admin/custom_emojis_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Admin
+  class CustomEmojisController < BaseController
+    def index
+      @custom_emojis = CustomEmoji.all
```
Comment (ykzts) [inline app/controllers/admin/custom_emojis_controller.rb]: Do you not paginate?

Comment (Gargron) [reply app/controllers/admin/custom_emojis_controller.rb]: It makes no sense to me to have more emojis than 40 or 50 at most, so no pagination required

That being said I realized this should filter by local ones only (for now)

Comment (ykzts) [reply app/controllers/admin/custom_emojis_controller.rb]: okay! and

> local ones only

Certainly.

**Review by ykzts — APPROVED** [review]

**Review by akihikodaki — APPROVED** [review]
Emojos everywhere.

File: app/controllers/admin/custom_emojis_controller.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Admin
+  class CustomEmojisController < BaseController
+    def index
+      @custom_emojis = CustomEmoji.where(uri: nil)
```
Comment (ykzts) [inline app/controllers/admin/custom_emojis_controller.rb]: Ah...

`uri: nil` 🙅‍♂️ 
`domain: nil` 🙆

**Pull request merged**